### DISCUSSION TITLE
ERROR: Failed to bind socket, port 8080 may already be in use.

### DISCUSSION THREAD

**Discussion by Ezherielll** [serverpod/serverpod#2181] [Q&A] [answered]
![image](https://github.com/serverpod/serverpod/assets/63665371/08bb1cd2-6de8-4d25-8be7-35a08591fc37)

When I run the command `dart bin/main.dart --apply-migrations` I get the error Failed to bind socket, port 8080 may already be in use. how do I solve this problem or do I need to replace the port and if so how to replace it?

**Comment by Ezherielll** [comment, marked as answer]
I have found the solution here : https://stackoverflow.com/questions/39632667/how-do-i-remove-the-process-currently-using-a-port-on-localhost-in-windows

**Comment by Metaildust** [comment]
There should be a container which is not related to your project is running in docker, so you need to shut it down, and then it will be solved
